moveit_planners/ompl/ompl_interface/src/parameterization/work_space/pose_model_state_space.cpp

```diff
@@ -54,6 +54,8 @@ const std::string PoseModelStateSpace::PARAMETERIZATION_TYPE = "PoseModel";
 
 PoseModelStateSpace::PoseModelStateSpace(const ModelBasedStateSpaceSpecification& spec) : ModelBasedStateSpace(spec)
 {
+  jump_factor_ = 1.5;  // \todo make this a param
```

Comment:
Totally optional since it's a forward port, but this should at least be a constexpr defined in an anonymous namespace up top. But of course, the TODO in the comment is preferred.

Reply:
This just restored old code. The variable is a member. Moving it into an anonymous namespace doesn't make sense.